### ISSUE TITLE
Accessibility Issue with Screenreader

### ISSUE THREAD

**Issue by sprungmarker** [jwplayer/jwplayer#3677]
### Expected Behavior

https://www.jwplayer.com/developers/web-player-demos/closed-captions/
After activating the great Play-Button with i.e. Screenreader JAWS in i.e. Firefox, the next Tab should reach the control area of the player i.e. de Play/Pause-Button.

### Actual Behavior

https://www.jwplayer.com/developers/web-player-demos/closed-captions/
After activating the great Play-Button with i.e. Screenreader JAWS in i.e. Firefox, the next Tab does not stay in the player, goes up to the browser tab.
The user has to tab to the player again and reaches the control area of the player i.e. de Play/Pause-Button.

### Steps to reproduce

https://www.jwplayer.com/developers/web-player-demos/closed-captions/
1. Tab with Screenreader to the player 
2. Activate the video with the great Play-Button
3. Tab with Screenreader

### Environment

- *Windows - Desktop*
- *Windows 10*
- *Firefox - 76; Edge 81*
- *JAWS- 2020, Windows Narrator*
- *https://www.jwplayer.com/developers/web-player-demos/closed-captions/*

----------------------------------------------------------------------------------------

**Comment by robwalch** [comment]
Similar to #3603

**Comment by robwalch** [comment]
Hi @sprungmarker - we've confirmed this issue internally and have slated it to be fixed in an upcoming patch.

This doesn’t have to do with the screen reader pre se. We can reproduce the issue with keyboard tabbing from the center play icon.

**Comment by aliciahurst** [comment]
Hi @sprungmarker, I'm the product manager of the player, and wanted to say that I'm sorry we're taking so long to reply to you. In fact, we recognize it often takes us _too long_ to reply here in github. I want you to get the help you need as fast as possible, which is why I'm going to close out this issue, and ask that if it still persists, you please [submit your issue](https://support.jwplayer.com/submit-support-case) to our fabulous technical support team, led by @SageBrushes. You'll be in great hands with them, and get a reply and support within a few business days. Thanks for submitting!
